hera_cal/datacontainer.py

```diff
@@ -10,7 +10,7 @@ def __init__(self, data):
             data (dict): dictionary of visibilities with keywords of pol/ant pair
                 in any order.
         """
-        self._data = {}
+        self._data = odict()
```

Comment:
This is fine, but if you input a non-ordered dictionary with length == 3 keys, line 24 will turn it back into a standard dict.